bridges/TwitterBridge.php

```diff
@@ -148,7 +148,7 @@ public function collectData(){
 			// extract fullname (pseudonym)
 			$item['fullname'] = $tweet->getAttribute('data-name');
 			// get author
-			$item['author'] = $item['fullname'] . ' (@' . $item['username'] . ')';
+			$item['author'] = htmlspecialchars_decode($item['fullname'] . ' (@' . $item['username'] . ')', ENT_QUOTES);
```

Comment:
That doesn't make sense.
`$item['fullname'] ` and `$item['username']` may still contain special chars, so they should be decoded beforehand.